app/controllers/users_controller.rb

```diff
@@ -1,6 +1,8 @@
 class UsersController < ApplicationController
   def show
     user = User.find(current_user.id)
+    @bookmarkings = user.bookmarkings(user)
```

Comment:
Hmm, ok we're going to need to mess with this cause I just had to change the user to an instance variable as well to get the token to be registered on the view page.

Reply:
We can at least leave the @user I have, and call @user.bookmarkings in the view. Ideally it would be nice if we didn't have a @user and a search_results being passed in though, I think.

Reply:
So why are we needing to call bookmarkings on a a user, and also pass it a user? Are they the same user, or no?